query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/wfs/persist/UpdateWfsConfigurationSpec.groovy

```diff
@@ -167,6 +165,38 @@ class UpdateWfsConfigurationTest extends Specification {
         report.messages().get(0).code == DefaultMessages.FAILED_PERSIST
     }
 
+    def 'Updating with flag password sends service properties without password'() {
+        setup:
+        updateWfsConfiguration.setValue(createWfsUpdateArgs(FLAG_PASSWORD))
+        serviceActions.read(_ as String) >> [(ID): TEST_SOURCENAME]
+        serviceReader.getServices(_, _) >> []
+        configurator.commit(_, _) >> mockReport(false)
+
+        def expectedUpdateConfig = getUpdateWfsProps(FLAG_PASSWORD)
```

Comment:
✏️ This test looks wrong, as it appears it's expecting the `FLAG_PASSWORD` will be sent as opposed to no password.